scripts/BuildAndTest.ps1

```diff
@@ -167,8 +167,6 @@ if (-not $NoRestore) {
 }
 
 if (-not $NoBuild) {
-    Update-VersionConstantsFiles
-    Update-VsixManifests
```

Comment:
>VsixManifests [](start = 11, length = 13)

Remove the functions since they are no longer called.

Reply:
Reactivating. As of iteration 10, these functions are still defined. Everything from lines 68-90 can go away.

---
In reply to: [521542752](https://github.com/microsoft/sarif-visualstudio-extension/pull/267#discussion_r521542752) [](ancestors = 521542752)

Reply:
just removed! :)

---
In reply to: [521587394](https://github.com/microsoft/sarif-visualstudio-extension/pull/267#discussion_r521587394) [](ancestors = 521587394,521542752)